src/metrics/metrics.py

```diff
@@ -13,32 +12,31 @@
     'gas_priority_fee_percentile',
     'min_priority_fee',
     'max_priority_fee',
-    'kafka_topic',
     'account_address',
     'create_transactions',
+    'modules_whitelist',
 ], namespace=PREFIX)
 
-GAS_FEE = Gauge('gas_fee', 'Gas fee', ['type'], namespace=PREFIX)
+GAS_FEE = Gauge('gas_fee', 'Gas fee', ['type', 'module_id'], namespace=PREFIX)
 
-DEPOSIT_FAILURE = Counter('deposit_failure', 'Deposit failure', namespace=PREFIX)
-SUCCESS_DEPOSIT = Counter('deposit_success', 'Deposit done', namespace=PREFIX)
+TX_SEND = Counter('transactions_send', 'Amount of send transaction from bot.', ['status'], namespace=PREFIX)
 
 ACCOUNT_BALANCE = Gauge('account_balance', 'Account balance', namespace=PREFIX)
 
-KAFKA_DEPOSIT_MESSAGES = Gauge(
-    'kafka_deposit_messages',
+DEPOSIT_MESSAGES = Gauge(
+    'deposit_messages',
     'Guardians deposit messages',
     ['address', 'version'],
     namespace=PREFIX,
 )
-KAFKA_PAUSE_MESSAGES = Gauge(
-    'kafka_pause_messages',
+PAUSE_MESSAGES = Gauge(
+    'pause_messages',
     'Guardians pause messages',
     ['address', 'version'],
```

Comment:
I think it's worth adding a label `module_id` for pause and deposit messages so we can separate them on the dashboard and have the details in the alert